resqpy/olio/xml_et.py

```diff
@@ -732,43 +715,29 @@ def _escaped_text(text):
         line += ' xmlns:' + type_pre_colon + '="' + ns[type_pre_colon] + '"'
         ns_keys.append(type_pre_colon)
 
-    for attrib in attrib_list:
-        line += ' ' + attrib
+    line += ' ' + ' '.join(attrib_list)
```

Comment:
if attrib_list is empty, this will add a trailing space (probably doesn't matter).

Reply:
Yeah good spot, I'll change it